lib/cache/cache_test.go

```diff
@@ -793,6 +793,62 @@ func benchListNodes(b *testing.B, nodeCount int, pageSize int) {
 	}
 }
 
+/*
+goos: linux
+goarch: amd64
+pkg: github.com/gravitational/teleport/lib/cache
+cpu: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
+BenchmarkListResourcesWithSort-8               1        2351035036 ns/op
+*/
+func BenchmarkListResourcesWithSort(b *testing.B) {
```

Comment:
hopefully i did this right, it was a replica of `benchGetNodes`: https://github.com/gravitational/teleport/blob/master/lib/cache/cache_test.go#L713

i ran this 10 times to get the average time